File: src/main/java/org/hyrulecraft/dungeon_utils/config/DungeonUtilsConfig.java
package org.hyrulecraft.dungeon_utils.config;

import eu.midnightdust.lib.config.MidnightConfig;

public class DungeonUtilsConfig extends MidnightConfig {

    @Entry(category = "settings") public static double revalisGaleHeight = 1.5;

    @Entry(category = "settings") public static boolean shouldAddSlowFalling = true;

    @Entry(category = "settings", min = 20.0, max = 1000.0) public static double hookShotRange = 20;
}

File: src/main/kotlin/org/hyrulecraft/dungeon_utils/config/DungeonUtilsConfig.kt
package org.hyrulecraft.dungeon_utils.config

import eu.midnightdust.lib.config.MidnightConfig

object DungeonUtilsConfig : MidnightConfig() {

    @JvmField
    @Entry(category = "settings") var revalisGaleHeight = 1.5

    @JvmField
    @Entry(category = "settings") var shouldAddSlowFalling = true

    @JvmField
    @Entry(category = "settings", min = 20.0, max = 1000.0) var hookShotRange = 20;
}